jekyll/_cci2/cci-academy-faqs.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: classic-docs
+title: CCI Academy FAQs
```

Comment:
I recommend expanding CCI to CircleCI throughout

Reply:
Done!